source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -65,18 +65,31 @@ Read more about [reporting](/reporting/#reporting) and [refunding payments](/ref
 
 ### When you'll receive payments
 
-You'll receive the money in your bank account within 2 working days of your user completing their payment.
+You'll receive the money in your bank account within 2 working days of your user completing their payment. It will take 3 working days if your user completed their payment at the weekend or on a bank holiday.
 
-It will take 3 working days if your user completed their payment at the weekend or on a bank holiday.
+Your payment service provider (PSP) will only transfer money to you when the total of your users' payments is above your PSP's ‘minimum payout’. For example if your PSP is Stripe, the minimum payout is £1.
 
-If your PSP is Stripe, you can sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) to see:
+If your PSP is Worldpay, SmartPay or ePDQ, ask your PSP if you need to know your exact minimum payout or payment times.
 
-- payments ('payouts') that Stripe has made to your bank account
-- which of your users' payments are in each payout
+#### Checking when your PSP sent a payment
 
-Your payment service provider (PSP) will only transfer money to you when the total of your users' payments is above your PSP's ‘minimum payout’. For example, if you use GOV.UK Pay’s PSP, the minimum payout is £1.
+If your PSP is Stripe, you can check when Stripe sent a payment to your bank account by making an API call to [get information about a payment](/reporting/#reporting-via-the-gov-uk-pay-api).
 
-If your PSP is Worldpay, SmartPay or ePDQ, ask your PSP if you need to know your exact minimum payout or payment times.
+Example response:
+
+```json
+"settlement_summary": {
+  ...
+  "psp_settled_date": "2016-01-22",
```

Comment:
```suggestion
  "settled_date": "2016-01-22",
```

suggest that we avoid the `psp_` prefix on fields in `settlement_summary`. The process of settlement is by definition something which is carried out by a PSP so it's redundant.

We already expose the following fields within `settlement_summary`:

```json
"settlement_summary": {
  "capture_submit_time": "2016-01-21T17:15:000Z",
  "captured_date": "2016-01-21"
}
```

these are described on the reporting page:

https://docs.payments.service.gov.uk/reporting/#checking-when-your-payment-service-provider-psp-took-a-payment

I think we should have one place where we describe the `settlement_summary`. Would it make more sense to consolidate the description of that functionality and cross reference it from here?